datalad/distributed/ora_remote.py

```diff
@@ -1097,27 +1098,21 @@ def _local_io(self):
         #       + just isinstance(LocalIO)?
         return not self.storage_host
 
+    # XXX this is pretty much obsolete, we should use self.message()
+    # which does debug messaging by default. however, I don't know if
+    # these prefixes are used for anything
```

Comment:
Re question in comment: No, prefix is not used anywhere, only exists to make it recognizable in a heavy debug output (datalad + annex +ORA).

Reply:
I removed it now. Thx!